lunavl/sdk/estimators/face_estimators/mask.py

```diff
@@ -143,27 +144,50 @@ class MaskEstimator(BaseEstimator):
     def __init__(self, maskEstimator: IMedicalMaskEstimatorPtr):
         """
         Init.
-
         Args:
             maskEstimator: core mask estimator
         """
         super().__init__(maskEstimator)
 
+    #  pylint: disable=W0221
+    @overload  # type: ignore
+    def estimate(self, image: FaceWarp) -> Mask:
+        """Estimate mask from a FaceWarp"""
+        pass
+
+    #  pylint: disable=W0221
+    @overload
+    def estimate(self, image: FaceWarpedImage) -> Mask:
+        """Estimate mask from a FaceWarpedImage"""
+        pass
+
+    #  pylint: disable=W0221
+    @overload
+    def estimate(self, image: VLImage, detection: DetectionFloat) -> Mask:
```

Comment:
why  detection has type  DetectionFloat? you can call detection.coreEstimation inside this function.  Core object  are not use in api lunasdk.

Reply:
Rewrote